cosim/black-parrot-example/tcl/bd.tcl

```diff
@@ -27,11 +27,11 @@ proc vivado_create_ip { args } {
     create_bd_port -dir O -type data tag_ck
     create_bd_port -dir O -type data tag_data
 
-    make_bd_intf_pins_external [get_bd_intf_pins top/s00_axi] -name "s00_axi"
-    make_bd_intf_pins_external [get_bd_intf_pins top/s01_axi] -name "s01_axi"
-    make_bd_intf_pins_external [get_bd_intf_pins top/s02_axi] -name "s02_axi"
-    make_bd_intf_pins_external [get_bd_intf_pins top/m00_axi] -name "m00_axi"
-    make_bd_intf_pins_external [get_bd_intf_pins top/m01_axi] -name "m01_axi"
+    make_bd_intf_pins_external [get_bd_intf_pins -hier s00_axi] -name "s00_axi"
+    make_bd_intf_pins_external [get_bd_intf_pins -hier s01_axi] -name "s01_axi"
+    make_bd_intf_pins_external [get_bd_intf_pins -hier s02_axi] -name "s02_axi"
+    make_bd_intf_pins_external [get_bd_intf_pins -hier m00_axi] -name "m00_axi"
+    make_bd_intf_pins_external [get_bd_intf_pins -hier m01_axi] -name "m01_axi"
```

Comment:
Is this required for a specific Vivado version? If so, which one(s) is it compatible or not compatible with?